packages/fxa-payments-server/src/routes/Product/index.tsx

```diff
@@ -97,20 +97,20 @@ export const Product = ({
     selectedPlan = productPlans[0];
   }
 
-  const onPayment = useCallback((tokenResponse: stripe.TokenResponse) => {
+  const onPayment = useCallback((tokenResponse: stripe.TokenResponse, name: string) => {
     if (tokenResponse && tokenResponse.token) {
       createSubscription(accessToken, {
         paymentToken: tokenResponse.token.id,
         planId: selectedPlan.plan_id,
-        displayName: profile.result ? profile.result.displayName : '',
+        displayName: name,
```

Comment:
For bonus points, if you name the param on line 100 as `displayName`, this line could just be `displayName,`

Reply:
There's another displayName used elsewhere in payments-server, so I thought this might be clearer, but idk.

Reply:
Yeah, not worth changing if it makes anything less clear :+1: